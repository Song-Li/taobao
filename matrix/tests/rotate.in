SET c = sequence 1 1
SET d = rotated c
SHOW d
