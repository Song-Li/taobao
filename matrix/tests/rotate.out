> ok

> ok

> 13.00 9.00 5.00 1.00
14.00 10.00 6.00 2.00
15.00 11.00 7.00 3.00
16.00 12.00 8.00 4.00

> 
bye
